GitUI/CommandsDialogs/SettingsDialog/ConfigFileSettingsPage.cs

```diff
@@ -24,23 +24,23 @@ protected override ISettingsSource GetCurrentSettings()
 
         public void SetEffectiveSettings()
         {
-            if (ConfigFileSettingsSet is not null)
+            if (ConfigFileSettingsSet.EffectiveSettings is not null)
```

Comment:
Could you please help me understand what changed that now ensures `ConfigFileSettingsSet` is not a null?
Should it be?
```suggestion
            if (ConfigFileSettingsSet?.EffectiveSettings is not null)
```

Reply:
I have turned `ConfigFileSettingsSet` into a `[record] struct`.

Reply:
Makes sense 👍